dll/win32/browseui/shellbars/CBandSite.cpp

```diff
@@ -694,14 +694,37 @@ HRESULT STDMETHODCALLTYPE CBandSiteBase::OnWinEvent(HWND hWnd, UINT uMsg, WPARAM
         
         return S_OK;
     }
-        
-    if (hWnd == fRebarWindow)
+    else if (uMsg == WM_COMMAND && lParam)
     {
-        /* FIXME: Just send the message? */
-        *plrResult = SendMessageW(hWnd, uMsg, wParam, lParam);
-        return S_OK;
+        hWnd = reinterpret_cast<HWND>(lParam);
+        goto forward_message;
```

Comment:
Not possible to make it a function instead of goto?

Reply:
Actually I can remove the goto and the behaviour will be the same.